.env

```diff
@@ -1,3 +1,8 @@
-REMOTE_DEV_DB=postgres://dkvnqdsh:HimRBU1zxUYm3Z60-gXlM2-u8lk-iwYp@pellefant.db.elephantsql.com:5432/dkvnqdsh
+NODE_ENV=test
```

Comment:
It's great that you created a `.env` file but why are you setting `NODE_ENV` in the `.env` file when `.env` usually should not be pushed to github. If `NODE_ENV` is to be set, wouldn't setting it in `package.json` file be better?

Reply:
Yeah, thanks for your review. The .env file shouldn't be pushed to GitHub as you've pointed out, but to make sure everyone on our team cloning the project gets up to speed in a short time, I decided to add it here since this is just for practice purposes. 

Setting it in package.json may not be a better option as this environment variable will also need to be set on Heroku. If it's not set, the express application always defaults to use 'development' as it's environment. Cheers.